Nodejs/Product/Nodejs/ProjectTemplates/AzureNodejsWorker/worker.js

```diff
@@ -4,6 +4,6 @@
  */
 'use strict';
 
-while (true) {
```

Comment:
This looks like a code smell to me. It's ok for now as nobody has complain about it.

Reply:
fair enough